bin/www

```diff
@@ -7,6 +7,13 @@
 var app = require('../app');
 var debug = require('debug')('group-project-faq:server');
 var http = require('http');
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+const MongoClient = require('mongodb').MongoClient;
+const Entry = require('../models/Entry');
+
+
+
```

Comment:
None of these variables appear to be used in `/bin/www` so you don't need to import them. Remove all this code.